documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,64 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Create a `GrafanaDatasource` CR so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
+
+* Create a Prometheus `ServiceMonitor` CR for your {brandname} cluster.
+* Install the Grafana operator and create a Grafana deployment.
+
+.Procedure
+
+. Create a service account for the Grafana data source.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/prometheus_service_account.yaml[]
+----
++
+.. Apply the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ {oc_apply_cr} service-account.yaml
+----
++
+.. Assign the `cluster-monitoring-view` role to the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc adm policy add-cluster-role-to-user cluster-monitoring-view -z monitoring-account
+----
++
+. Export required system details to environment variables.
+.. Retrieve an authentication token for the service account.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ export BEARER_TOKEN=$(oc serviceaccounts get-token monitoring-account)
```

Comment:
This also shouldn't be necessary and appears to be very specific to a certain environment.